plugins/katello/nightly/user_guide/remote_databases/index.md

```diff
@@ -0,0 +1,270 @@
+Enable authentication in `/etc/mongod.conf`
+```
+auth=true
+```
+
+Enable and start the service
+```
+  systemctl start mongod
+  systemctl enable mongod
+```
+
+### Create Pulp user and database 
+```
+  mongo admin -u admin -p admin --eval "db.createUser({user:'pulp',pwd:'<PULP_PASSWORD>',roles:[{role:'dbOwner', db:'pulp_database'},{ role: 'readWrite', db: 'pulp_database'}]})"
+
+```
+
+### Test it works
+From `katello.example.com` test the mongo DB is accessible:
+```
+  mongo --host mongo.example.com -u pulp -p <PULP_PASSWORD> --port 27017 --eval 'ping:1' pulp_database
+```
+If there are no errors we are done with database preparation. 
+
+## Fresh install
+
+### Install katello package
+We assume the box where the Katello server will be installed has hostname `katello.example.com`.
+
+Follow the [documentation](plugins/katello/{{ page.version }}/installation/index.html) to install the `katello` package and **do not run foreman-installer.** We need to use the remote database flags with the installer. Use the following steps once the katello rpm is installed.
+
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for installation.
+
+### Run the installer
+To install and configure Katello we just need to run 
+```
+  foreman-installer --scenario katello \
+    --foreman-db-host postgres.example.com \ 
+    --foreman-db-password <FOREMAN_PASSWORD> \
+    --foreman-db-database foreman \
+    --katello-candlepin-db-host postgres.example.com \
+    --katello-candlepin-db-name candlepin \
+    --katello-candlepin-db-password <CANDLEPN_PASSWORD> \
+    --katello-candlepin-manage-db false \
+    --katello-pulp-db-username pulp \ 
+    --katello-pulp-db-password <PULP_PASSWORD> \ 
+    --katello-pulp-db-seeds “mongo.example.com:27017” \
+    --katello-pulp-db-name pulp_database
+```
+
+Note: for more related options and tips on SSL configuration see [Full list of options](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#full-list-of-remote-database-related-options-in-the-installer)
+
+## Migration of existing Katello
+We assume that Katello was installed and is running on `katello.example.com`.
+
+### Prepare remote databases
+Follow the instructions to [prepare remote mongo](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-mongo) and [prepare remote postgres](plugins/katello/{{ page.version }}/user_guide/remote_databases/index.html#prepare-remote-postgres) to make the remote database servers ready for migration.
+
+### Stop the Katello server
+Stop the Katello related services to minimize risk of the data changes during the migration
+```
+  katello-service stop
+  systemctl start postgresql
```

Comment:
You can also use `systemctl start postgresql mongodb` so it happens in parallel or `katello-service stop --exclude postgresql,mongod` to prevent them from stopping in the first place.

Reply:
thanks, will keep them as separate steps in case they are stopped for some reason